packages/jest-runtime/src/index.ts

```diff
@@ -2123,6 +2127,15 @@ export default class Runtime {
           );
         }
       },
+      getSeed: () => {
+        // TODO: remove this check in Jest 30
+        if (this._globalConfig?.seed === undefined) {
+          throw new Error(
+            'The seed value is not available. Likely you are using older versions of the jest dependencies.',
+          );
+        }
+        return this._globalConfig?.seed;
```

Comment:
```suggestion
        return this._globalConfig.seed;
```

If TS disagrees, extract to a variable before checking for undefined